omnigibson/systems/system_base.py

```diff
@@ -1233,6 +1233,10 @@ def is_physical_particle_system(system_name):
 def get_system(system_name, force_active=True):
     # Make sure scene exists
     assert og.sim.scene is not None, "Cannot get systems until scene is imported!"
+    # Make sure prefixes preserve their double underscore
+    for prefix in SYSTEM_PREFIXES:
+        if f"{prefix}__" not in system_name:
+            system_name = system_name.replace(f"{prefix}_", f"{prefix}__")
```

Comment:
This  is a bug fix with @ChengshuLi , we have to correctly handle the transition from single underscore category to double underscore synset

Reply:
removed